client/src/components/App/Content/InvestigationForm/TabManagement/InteractionsTab/InteractionDialog/InteractionFormTabSwitchButton.tsx

```diff
@@ -0,0 +1,63 @@
+import React from 'react';
+import {useFormContext} from 'react-hook-form';
+import {Button, Tooltip} from '@material-ui/core';
+import {ChevronLeft} from '@material-ui/icons';
+import InteractionEventDialogFields
+    from 'models/enums/InteractionsEventDialogContext/InteractionEventDialogFields';
+import {get} from 'Utils/auxiliaryFunctions/auxiliaryFunctions';
+
+const cantMoveToContactsBaseMessage = 'לא ניתן לעבור ליצירת מגעים';
+
+interface ContinueToTabBaseType {
+    canContinue: boolean;
+}
+
+interface ContinueToTabOK extends ContinueToTabBaseType {
+    canContinue: true;
+}
+
+interface ContinueToTabError extends ContinueToTabBaseType {
+    canContinue: false;
+    message: string;
+}
```

Comment:
I see that those interfaces are extending ContinueToTabBaseType but they're just overriding it, why extend it then? 🤔

Reply:
to establish the common structure

Reply:
OK

Reply:
OK